meta-fobnail-distro/recipes-fobnail/packagegroups/packagegroup-fobnail.bb

```diff
@@ -13,4 +13,5 @@ RDEPENDS:${PN}-system = " \
   packagegroup-core-base-utils \
   chrony \
   chronyc \
+  secure-kernel-loader \
```

Comment:
@kotylamichal is this package needed in the system? What does it install? Don't we need only

```
do_image_wic[depends] += " \
  secure-kernel-loader:do_deploy \
"
```